x-pack/plugins/security_solution/cypress/tasks/common.ts

```diff
@@ -280,3 +281,8 @@ export const waitForWelcomePanelToBeLoaded = () => {
   cy.get(KIBANA_LOADING_ICON).should('exist');
   cy.get(KIBANA_LOADING_ICON).should('not.exist');
 };
+
+export const waitForTableToLoad = () => {
+  cy.get(EUI_BASIC_TABLE_LOADING).should('exist');
+  cy.get(EUI_BASIC_TABLE_LOADING).should('not.exist');
```

Comment:
Is this `EUI_BASIC_TABLE_LOADING` locator used for all the tables in the solution?

Reply:
It matches `EUIBasicTable` https://elastic.github.io/eui/#/tabular-content/tables#a-basic-table, which is widely used in the solution.

Reply:
great! then good move! thanks :D